.travis.yml

```diff
@@ -30,7 +30,7 @@ env:
         - MAIN_CMD='python setup.py'
         - SETUP_CMD='test'
         - EVENT_TYPE='pull_request push'
-        - PIP_DEPENDENCIES=''
+        - PIP_DEPENDENCIES='gwcs'
```

Comment:
It doesn't matter much how it's installed being a small package, but we build conda packages for ``gwcs`` in the astropy channel if you ever need it.